src/TypeSpec.Extension/Emitter.Csharp/src/lib/operation.ts

```diff
@@ -101,11 +102,17 @@ export function loadOperation(
         parameters.push(loadOperationParameter(sdkContext, p));
     }
 
-    if (typespecParameters.body?.parameter) {
+    if (
+        typespecParameters.body?.parameter &&
+        !isVoidBody(typespecParameters.body.type)
+    ) {
         parameters.push(
             loadBodyParameter(sdkContext, typespecParameters.body?.parameter)
         );
-    } else if (typespecParameters.body?.type) {
+    } else if (
+        typespecParameters.body?.type &&
+        !isVoidBody(typespecParameters.body.type)
```

Comment:
will this affect DPG?

Reply:
Yes, I think for DPG, a void body means no body too.